src/app.js

```diff
@@ -1 +1,26 @@
+/* eslint-disable no-console */
 'use strict';
+
+const fs = require('fs');
+
+function copyFile() {
+  const [from, to] = process.argv.slice(2);
+
+  if (from === to) {
+    console.error('error');
+
+    return;
+  }
+
+  if (!from || !to) {
+    return console.error('error');
+  }
+
+  fs.cp(from, to, (error) => {
```

Comment:
The `fs.cp` method is not available in Node.js. You should use `fs.copyFile` instead. The correct usage would be `fs.copyFile(from, to, (error) => {...})`.